cookiecutter.json

```diff
@@ -5,12 +5,12 @@
     "author": "Jane Doe",
     "author_email": "jane_doe@imperial.ac.uk",
     "rse_team_as_coauthor": false,
-    "use_bsd3_licence": false,
+    "use_bsd3_license": true,
```

Comment:
I'm not sure we should make projects open source by default, because I'd worry that a random researcher might just choose the default here without understanding what they were saying yes to.

Licence is the UK spelling, but I guess it doesn't matter much. We call the file LICENSE not LICENCE anyway (I think GitHub is happy with either).

Reply:
> I'm not sure we should make projects open source by default, because I'd worry that a random researcher might just choose the default here without understanding what they were saying yes to.

I guess since this repo is under the Imperial org we should follow Imperial guidelines and use/encourage use of this licence? Even if they don't make the repo open source they might invite private collaborators who should follow the terms of any licence.

> Licence is the UK spelling, but I guess it doesn't matter much. We call the file LICENSE not LICENCE anyway (I think GitHub is happy with either).

Ew, just realised I've been using US spelling everywhere 😫

Reply:
> I'm not sure we should make projects open source by default, because I'd worry that a random researcher might just choose the default here without understanding what they were saying yes to.

True, I only made this a default because I wanted to test that the pyproject.toml was formatted correctly with the addition of the license sections. I can make it false by default and then include it as a command line argument in CI instead.

> Licence is the UK spelling, but I guess it doesn't matter much. We call the file LICENSE not LICENCE anyway (I think GitHub is happy with either).

I actually didn't realise that. Maybe that's why they both look wrong to me! I just wanted consistency since all the other uses in the codebase are "License"